packages/melos/lib/src/command_runner/format.dart

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2016-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
```

Comment:
I don't think this licence header is needed, it seems very arbitrary which files that have it and which don't.
@Salakar could all these be removed, since we have the licence file anyways? If so, I can do it in a follow-up for the rest.